src/main/scala/com/ambiata/disorder/NaturalIntSmall.scala

```diff
@@ -8,7 +8,7 @@ case class NaturalIntSmall(value: Int)
 object NaturalIntSmall {
 
   implicit def NaturalIntSmallArbitrary: Arbitrary[NaturalIntSmall] =
-    Arbitrary(Gen.sized(n => Gen.choose(1, math.max(1, n))).map(NaturalIntSmall.apply))
+    Arbitrary(Gen.sized(n => Gen.choose(0, math.max(0, n))).map(NaturalIntSmall.apply))
```

Comment:
FYI

    Gen.sized(n => Gen.choose(x, math.max(x, n)))

Can be replaced with

    GenPlus.choose(x, Int.maxValue)